### DISCUSSION TITLE
Selecting a default sys-language and sys-encoding

### DISCUSSION THREAD

**Discussion by lavafroth** [archlinux/archinstall#1654] [Q&A] [answered]
I might be wrong in how this works but on this [line](https://github.com/archlinux/archinstall/blob/master/archinstall/__init__.py#L224) we are trying to set a default value for 'sys-language' after making sure that it is not `None`. Can we try something like

```py
arguments['sys-language'] = arguments.get('sys-language', 'en_US')
```

without the if condition?

**Comment by Torxed** [comment, marked as answer]
I don't see why not. Nice catch!
